php.html.markdown

```diff
@@ -426,6 +426,17 @@ echo $function_name(1, 2); // => 3
 // Useful for programatically determining which function to run.
 // Or, use call_user_func(callable $callback [, $parameter [, ... ]]);
 
+// You can specify the type of variable a function parameter accepts using type hints:
```

Comment:
please add a note about the impossibility to use scalars (except for arrays and I guess Callable, and until 7.0.0 I guess...)